app/Http/Controllers/Item/ItemController.php

```diff
@@ -26,8 +26,8 @@ public function index()
         // TO-DO
         // Ask fro help from Jeseph of how to use the config values on pagination
         // get the pagination number or a default
-        $items = $this->$itemRepository ?? 10;
-        return view('items' . config('settings.pagination.small') . '.index')->with(['$items' => $items]);
+        $items = $this->$itemRepository->paginate(config('settings.pagination.small')) ?? 10;
```

Comment:
```suggestion
        $items = $this->$itemRepository->paginate(config('settings.pagination.small'));
```

Previously you could end up with `$items = 10`; But questions is, where are you getting the number 10 from? By having the value coming from the `settings.pagination.small` means you are already setting a default value.